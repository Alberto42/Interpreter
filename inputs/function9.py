def f(x) {
    return x + "sufix"
}

result = f("prefix ")