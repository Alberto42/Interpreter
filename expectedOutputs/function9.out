Right (OK,
Ident "result" = VString "prefix sufix"
)
